tests/dcerpc/dcerpc-dce-iface-many/test.rules

```diff
@@ -0,0 +1 @@
+alert dcerpc any any -> any any (msg: "DCE Netlogon"; dce_iface: 12345678-1234-abcd-ef00-01234567cffb; sid: 1;)
```

Comment:
Not sure if it matters for the test but is it worth doing it with `dcerpc.iface` and `dcerpc.opnum` keywords (respectively below) instead as those are the new/sticky buffer stanza ?

Reply:
indeed. Thanks, Peter!